docs/recipes/lighthouse-plugin-example/readme.md

```diff
@@ -7,8 +7,8 @@
  
 ## To run
 
-- Install as a (peer) dependency, parallel to `lighthouse`.
-- `npx lighthouse https://example.com --plugins=lighthouse-plugin-example --view`
+1. Install the plugin as a (peer) dependency, parallel to `lighthouse`.
+2. Verify that the lighthouse version installed supports plugins >= 5.20 and run the following command `npx lighthouse https://example.com --plugins=lighthouse-plugin-example --view`
```

Comment:
I'm not quite sure what this sentence is saying. Maybe...
```suggestion
1. Install `lighthouse` v5 or later.
2. Install the plugin as a (peer) dependency, parallel to `lighthouse`.
3. Run `npx lighthouse https://example.com --plugins=lighthouse-plugin-example --view`
```

if you're just trying to capture that you need a compatible lh version?

Reply:
I'm also not entirely sure what the focus on 5.2.0 is, I don't recall shipping any particular plugin changes in that version do you @brendankenny?

Reply:
also should the run command be `npx -p lighthouse lighthouse https://example.com ...` based on https://github.com/zkat/npx/issues/215?